whats_fresh/whats_fresh_api/views/entry/vendors.py

```diff
@@ -219,7 +220,7 @@ def vendor_list(request):
         'message': message,
         'new_url': reverse('new-vendor'),
         'new_text': "New Vendor",
-        'title': "All Vendors",
+        'title': get_fieldname.get_fieldname('vendors', 'vendors'),
```

Comment:
get_fielname() takes 2 arguments, in list.html I am using `{{ title }}` just as before.